assignments/arrays.js

```diff
@@ -65,30 +65,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+function findID(id){
+    for (let i = 0; i < inventory.length; i++){
+      if (inventory[i].id === id){
+        console.log('Car' + id +' is a ' + inventory[i].car_year + inventory [i].car_make + ' ' + inventory[i].car_model);
+      }
+    }
+   }  
```

Comment:
The loop continues to run even after the if statement passes and logs to the console. There's no need to continue the loop afterwards since it just uses up unnecessary resources. To "break" out of the loop, you could use the `break` keyword right after the console log:
```js
if (inventory[i].id === id) {
  console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`);
  break;
}
```